overlays/haskell.nix

```diff
@@ -471,7 +471,15 @@ final: prev: {
                   inherit (rev) sha256;
                 };
                 revSuffix = final.lib.optionalString (rev.revNum > 0) "-r${toString rev.revNum}";
-            in let src = final.buildPackages.pkgs.runCommand "${name}-${version'}${revSuffix}-src" { } (''
+            in let src = final.buildPackages.pkgs.runCommand "${name}-${version'}${revSuffix}-src" {
+                passthru = {
```

Comment:
I assume this does the right thing, but in this instance I'd probably just override the `filterPath` attribute with `//` afterwards!

Reply:
I have switched it to `//`. I don't really know when `passthru` is better than `//`.  I guess maybe you can `override` `passthru`?

Reply:
I think `passthru` is useful when you don't have access to the result of a function (e.g. because it's being called inside something else). But here you do.